identity-vault/startup-strategies/src/pages/Start.tsx

```diff
@@ -0,0 +1,20 @@
+import { useHistory } from 'react-router-dom';
+import { isAuthenticated } from '../util/authentication';
+
+const Start = () => {
+  const history = useHistory();
+
+  const performNavigation = async () => {
+    if (await isAuthenticated()) {
+      history.replace('/tabs/tab1');
+    } else {
+      history.replace('/login');
+    }
+  };
+
+  performNavigation().catch(() => history.replace('/unlock'));
```

Comment:
Given the fact that this FC will run a grand total of once at startup and will not  ever re-render I figured it was best to avoid the overhead of a `useEffect()` even though we are syncing with the vault here.

Reply:
Side note, this makes me want to simplify the Vue and Angular Start pages. I may look at that next.